packages/preset-react-app/docs/getting-started/1.md

```diff
@@ -11,9 +11,11 @@ These lines will install Phenomic and its [react app preset](https://github.com/
 
 ⚠️ _If you don't have a ``package.json`` you can do this simple command to get
  a valid one:_
- 
+
 ```console
 echo '{"private":true}' > package.json
+# or you can also do
+yarn init
```

Comment:
I guess we should also specify `npm init` then? Just afraid this create more noize